dts/bindings/ieee802154/nxp,mcxw-ieee802154.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP MCXW71 IEEE 802.15.4 node
+
+compatible: "nxp,mcxw-ieee802154"
```

Comment:
similar comment, do not put mcxw in compatible string, in case this radio IP is also on another part, is there a name of the IP?

Reply:
Currently the SDK Phy API that is used by this driver only supports two boards MCXW71 and MCXW72 so I believe that this name is ok in this case

Reply:
we really have a hard rule not to put soc names in the IP compatible strings. for now i will not block this PR merging over it, since it's not a node a board would define, but really needs to be updated later

Reply:
what do you propose?
We can update it when introducing the mcxw72 ieee 802.15.4 driver?

Reply:
The fw runs on the NBU core. You can't access directly the radio hw